public/app/features/dashboard/components/DashboardSettings/LinksSettings.tsx

```diff
@@ -11,24 +13,46 @@ export type LinkSettingsMode = 'list' | 'new' | 'edit';
 
 export function LinksSettings({ dashboard, sectionNav }: SettingsPageProps) {
   const [editIdx, setEditIdx] = useState<number | null>(null);
+  const [isNew, setIsNew] = useState<boolean>(false);
+  const { chrome } = useGrafana();
+
+  React.useEffect(() => {
+    if (sectionNav) {
+      chrome.update({
+        sectionNav: sectionNav.node,
+      });
+    }
+  }, [sectionNav, chrome]);
```

Comment:
is this necessary? i think `Page` already does this 🤔

Reply:
Yes, you're right. Before adding the `pageNav` in the `New link` and `Edit link` pages, I saw 'Link' twice in the `Breadcrumbs`and I instantly remember the solution of the `Explore` page. I'll get rid of it. Thanks!